project/plugins.sbt

```diff
@@ -1,12 +1,14 @@
 // keep in sync with the play version in Dependencies
-addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.1")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.2")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-coffeescript" % "1.0.2")
-addSbtPlugin("com.typesafe.sbt" % "sbt-less" % "1.1.2")
-addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
+addSbtPlugin("com.github.sbt" % "sbt-coffeescript" % "1.11.0")
+addSbtPlugin(
+  "com.typesafe.sbt" % "sbt-less" % "1.1.2"
+) // scala-steward:off --  upgrading to 1.5 is causing a hard to debug error about mkdrip not existing.
```

Comment:
CI kept failing due to a `mkdirp not found` error. Tried really hard to replicate it locally but wasnt able to. Going to skip this bump for now in order to get the rest of the dependency bumps out.